### ISSUE TITLE
Disable Grid view in Media Library

### ISSUE THREAD

**Issue by oksana-c** [department-of-veterans-affairs/va.gov-cms#3607]
## Description

According to Nielsen's Usability Heuristics, the Grid View, that is available as a part of the CMS media library, violates the following heuristics: Accessibility, Error prevention, Consistency and standards, Consistency and standards, User control and freedom, Recognition rather than recall.

Note from Heuristic audit: 
> Table and Grid Views are not only different views - they have different content including : 1) Action drop down menu not present in Table, 2) Intro text is not present in view, 3) Size not included in Table view

Additionally, the Grid view offers Actions dropdown while, while follows existing user permissions, allows for certain irreversible actions to be performed in bulk. These bulk Actions may introduce undesirable consequences for existing data, 
they were not specifically requested by users and did not go thru design validation.

<img width="1121" alt="Screen Shot 2020-12-01 at 2 51 55 PM" src="https://user-images.githubusercontent.com/16477724/100801163-d855e500-33e4-11eb-9ec4-a03f1a025dc5.png">

## Recommendation

Disable Grid view in media library.

## Acceptance criteria

- [ ] the Grid view is disabled (not removed) in the Media Library View - /admin/structure/views/view/media_library/edit/page
